transport.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'constants'
```

Comment:
Also I haven't found a comparison of Bike and Car instances like said here:

> расширить класс Transport чтобы объекты этого класса можно было сравнивать по скорости доставки 1 кг посылки

Reply:
I added this method but don't use. I'm not sure where I can use it